google/cloud/grpc_utils/version.h

```diff
@@ -15,7 +15,14 @@
 #ifndef GOOGLE_CLOUD_CPP_GOOGLE_CLOUD_GRPC_UTILS_VERSION_H
 #define GOOGLE_CLOUD_CPP_GOOGLE_CLOUD_GRPC_UTILS_VERSION_H
 
+#include "google/cloud/internal/attributes.h"
```

Comment:
is this needed?

Reply:
Nope, I thought I would use `[[deprecated]]` attributes, but at the end I did not.